app/components/calendar-page.js

```diff
@@ -9,7 +9,7 @@ export default Component.extend({
     day: computed("date", function () {
         const date = get(this, "date");
 
-        return leftPad("00", new Date(date).getDate().toString());
+        return leftPad("00", new Date(date).getUTCDate().toString());
```

Comment:
Good catch :+1: , I think the same should be done for month few lines below (`getUTCMonth`).